src/PathTracedVolume.ts

```diff
@@ -354,6 +354,14 @@ export default class PathTracedVolume {
     // this.visible = isVisible;
   }
 
+  public setShowBoundingBox(_showBoundingBox: boolean): void {
+    // TODO: NOT IMPLEMENTED YET
+  }
+
+  public setBoundingBoxColor(_color: [number, number, number]): void {
+    // TODO: NOT IMPLEMENTED YET
```

Comment:
Are these functions needed (part of the API) or just set as reminders?

Reply:
Yep, they are needed as interface methods.